devices/lixee.js

```diff
@@ -688,4 +755,25 @@ const definition = {
     },
 };
 
-module.exports = [definition];
+const definitionZiPulses = {
+    zigbeeModel: ['ZiPulses'],
+    model: 'ZiPulses',
+    vendor: 'LiXee',
+    description: 'Lixee ZiPulses',
+    fromZigbee: [fz.battery, fz.temperature, fz.metering, fzZiPulses],
+    toZigbee: [tzSeMetering],
+    exposes: [e.battery_voltage(), e.temperature(),
+        exposes.numeric('multiplier', ea.STATE_SET).withValueMin(1).withValueMax(1000).withDescription('It is necessary to press the link button to update'),
+        exposes.numeric('divisor', ea.STATE_SET).withValueMin(1).withValueMax(1000).withDescription('It is necessary to press the link button to update'),
+        exposes.enum('unitOfMeasure', ea.STATE_SET, unitsZiPulses).withDescription('It is necessary to press the link button to update'),
+        exposes.numeric('energy', ea.STATE),
+    ],
+    configure: async (device, coordinatorEndpoint, logger) => {
+        const endpoint = device.getEndpoint(1);
+        const binds = ['genPowerCfg', 'seMetering', 'msTemperatureMeasurement'];
+        await reporting.bind(endpoint, coordinatorEndpoint, binds);
+        await endpoint.read('seMetering', ['divisor', 'unitOfMeasure', 'multiplier']);
+    },
+};
+
+module.exports = [definitionZlinky, definitionZiPulses];
```

Comment:
Please inline the definitions (like e.g. https://github.com/Koenkk/zigbee-herdsman-converters/blob/2e2c21f08916fc17727a108adc2306d952c53288/devices/awox.js#L63)